### PR TITLE
Refactor service transport startup

### PR DESCRIPTION
**Pull request by artem-v**

### CODE REVIEW

File: services-transport-rsocket/src/main/java/io/scalecube/services/transport/rsocket/RSocketServiceTransport.java [hunk 1]
```diff
@@ -1,81 +1,42 @@
...
+import reactor.netty.resources.LoopResources;
 
 /**
  * RSocket service transport. Entry point for getting {@link RSocketClientTransport} and {@link
  * RSocketServerTransport}.
  */
 public class RSocketServiceTransport implements ServiceTransport {
 
-  private static final String DEFAULT_HEADERS_FORMAT = "application/json";
+  public static final RSocketServiceTransport INSTANCE = new RSocketServiceTransport();
```
Comment (ronenhamias) [inline services-transport-rsocket/src/main/java/io/scalecube/services/transport/rsocket/RSocketServiceTransport.java:17]: we really need this INSTANCE?

**Comment by ronenhamias** [comment]
need to update docs how to use

**Comment by ronenhamias** [comment]
need to update readme file

**Review by segabriel — APPROVED** [review]
looks cool

**Comment by artem-v** [comment]
> need to update docs how to use
> need to update readme file

Will do after finish  Ronen's comments which concerns having better  decoupled  code in  `ScalecubeServiceDiscovery` -- https://github.com/scalecube/scalecube-services/issues/519

**Pull request merged**